mazerunner-scenarios/src/main/java/com/bugsnag/android/mazerunner/TestCaseFactory.kt

```diff
@@ -4,7 +4,7 @@ import android.content.Context
 import com.bugsnag.android.Configuration
 import com.bugsnag.android.mazerunner.scenarios.Scenario
 
-internal class TestCaseFactory {
+class TestCaseFactory {
```

Comment:
This is required to make `TestCaseFactory` visible to other modules which depend on `mazerunner-scenarios`